compiler/compiler.cpp

```diff
@@ -312,6 +314,13 @@ bool compiler_execute(CompilerSettings *settings) {
     stage::error();
   }
 
+  if (!PerformanceIssuesReport::get().empty()) {
+    if (auto *report_file = fopen(G->settings().performance_analyze_report_path.get().c_str(), "w")) {
+      PerformanceIssuesReport::get().flush_to(report_file);
+      fclose(report_file);
+    }
+  }
```

Comment:
If there are no issues, the .json file is not re-created, so it remains an old one. 
Here is the situation:
1) I mark a function with @kphp-analyze-performance and see a report with several warnings
2) I fix all errors and execute KPHP again
3) The file remains — I suppose it's confusing

I'd suggest to create an empty file (just '[\n]') in case of any errors. (if there is a function with @kphp-analyze-performance of course)